ports/qemu-riscv/Makefile

```diff
@@ -0,0 +1,120 @@
+include ../../py/mkenv.mk
+-include mpconfigport.mk
+
+# qstr definitions (must come before including py.mk)
+QSTR_DEFS = qstrdefsport.h
+
+# MicroPython feature configurations
+MICROPY_ROM_TEXT_COMPRESSION ?= 1
+
+# include py core make definitions
+include $(TOP)/py/py.mk
+include $(TOP)/extmod/extmod.mk
+
+CROSS_COMPILE ?= riscv64-unknown-elf-
+
+BOARD ?= virt
+LDFLAGS = --gc-sections -Map=$(@:.elf=.map)
+LIBS = $(shell $(CC) $(CFLAGS) -print-libgcc-file-name)
```

Comment:
These LDFLAGS and LIBS line need to come later on in the Makefile, after all CFLAGS have been set.  Otherwise the LIBS command doesn't have all the correct CFLAGS.

I'd suggest putting them before the SRC_COMMON_C definition, following qemu-arm.

Reply:
Yep, that's where they were but right now I don't really remember the reason for moving them around - I'll address that.